services/scan/src/server.ts

```diff
@@ -67,19 +67,20 @@ export async function start({
         'workspace path could not be determined; pass a workspace or run with SCAN_WORKSPACE'
       );
     }
-    resolvedWorkspace = await createWorkspace(workspacePath);
+    resolvedWorkspace = createWorkspace(workspacePath);
   }
 
   let resolvedApp: express.Application;
 
   if (machineType === 'precinct-scanner') {
     const precinctScannerInterpreter = createInterpreter();
-    const precinctScannerMachine = createPrecinctScannerStateMachine(
+    const precinctScannerMachine = createPrecinctScannerStateMachine({
       createPlustekClient,
-      resolvedWorkspace.store,
-      precinctScannerInterpreter,
-      logger
-    );
+      scannedImagesPath: resolvedWorkspace.scannedImagesPath,
+      store: resolvedWorkspace.store,
```

Comment:
maybe just pass in the whole workspace since the store and scannedImagesPath will always come from workspace